cpp_test/videoFeed.cpp

```diff
@@ -0,0 +1,52 @@
+
+#include <opencv2/opencv.hpp>
+#include <iostream>
+
+using namespace cv;
+using namespace std;
+
+int main(int argc, char *argv[])
+{
+    //open the video file for reading
+    VideoCapture cap(0);
+
+    // if not success, exit program
+    if (cap.isOpened() == false)
+    {
+        cout << "Cannot open the video file" << endl;
+        cin.get(); //wait for any key press
+        return -1;
+    }
+
+    //Define names of the window
+    String windowName = "Video Feed";
+
+    // Create a window with above names
+    namedWindow(windowName, WINDOW_NORMAL);
+
+    while (true)
+    {
+        Mat frame;
+        bool bSuccess = cap.read(frame); // read a new frame from video
+        if (bSuccess == false)
```

Comment:
You should choose a way to do this and be consistent. Either always declare a variable or never declare one. Compare this line 31 to line 14 above. Same logic, same code pattern, different implementation